src/stirling/source_connectors/tcp_stats/bcc_bpf_intf/tcp_stats.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "src/stirling/upid/upid.h"
+
+#define MAX_CMD_SIZE 32
+
+union sockaddress_t {
+  struct sockaddr sa;
+  struct sockaddr_in in4;
+  struct sockaddr_in6 in6;
+};
+
+struct ip_key_t {
+  // The process name of this process
+  char name[MAX_CMD_SIZE];
+
+  // IP address of the remote endpoint.
+  union sockaddress_t addr;
+};
```

Comment:
Did not see this used. If it is dead code, we can remove it.

Reply:
Done